### DISCUSSION TITLE
Is it possible to install Docker on Termux Ubuntu then run it on VS codeserver?

### DISCUSSION THREAD

**Discussion by TheMightyArie** [coder/code-server#6871] [Q&A]
Hello! I'm not sure if it's right place to ask, but I wanted to ask something. If I install Docker to Termux with Ubuntu 24.04, am I able to run the Docker extension on VS codeserver? If you want me to ask this to Docker or Termux support, no worries.

**Comment by code-asher** [comment]
Which Docker extension?  Most of them should work, I think, except for the dev container one.
